client/src/components/squids/SquidSign.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+
+import { format, getDate, parseISO } from "date-fns";
+import signTeller from "sign-teller";
+
+import { SignShape } from "../../types/SignShape";
+
+export const SquidSign: FC<{ birthday: string }> = ({ birthday }) => {
+  const squidSign = (birthday: string): SignShape => {
+    const parsedDate = parseISO(birthday);
```

Comment:
It would be great to move your date-parsing logic into a helper method (particularly since you use it in multiple places) and then fire up a quick Jest test for that helper method :)

Reply:
do you mean for using `parseISO` with the birthday?

Reply:
I was thinking of a helper that takes in a birthday string and returns the day and month -- so, capturing the logic spanning line 10 to line 13. Without that, your useGreetings hook might be a little sparse, but... probably good practice to have a custom hook anyways :)